src/Tribe/Views/V2/View.php

```diff
@@ -2145,4 +2154,92 @@ protected function get_container_data() {
 
 		return $data;
 	}
+
+	protected function should_show_recent_past_events_view() {
```

Comment:
:no_entry_sign: Missing doc comment for function should_show_recent_past_events_view()

<sub>hash: 4099bfca1befd9c496d5f51ac494a896</sub>